src/engine.ts

```diff
@@ -12,11 +12,20 @@ import {
   Planet,
   Round,
   Booster,
-  Resource
+  Resource,
+  TechTile,
+  TechTilePos,
+  AdvTechTile,
+  AdvTechTilePos,
+  Federation
 } from './enums';
 import { CubeCoordinates } from 'hexagrid';
+import Event from './events';
+import techs from './tiles/techs';
+import researchTracks from './research-tracks'
```

Comment:
It's `import * as researchTracks from './research-tracks`, since there's no default exports but instead two separate functoins.

Reply:
for some reason when I tried to use this import it was giving me strange errors, so I decided to use a constant to speed up the process.. BTW, is there a specific reason to use a function instead of a constant?